Formalisation(shacl)/Core/PiecesShape/DataService.ttl

```diff
@@ -0,0 +1,41 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix vcard: <http://www.w3.org/2006/vcard#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix core: <http://coreRule-healthri.nl#> .
+@prefix : <http://coreRule-healthri.nl#> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
```

Comment:
Lot of unused imports; please reduce to the actual used namespaces to improve readability and reduce maintenance.

Reply:
done